news/2022-08-25-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -0,0 +1,143 @@
+
+Performance points in osu!taiko are currently calculated by summing the "accuracy pp" and "difficulty pp" components of a score — the following changes should serve to increase their values and improve balance not only between difficulty-increasing mods, but their reductive counterparts.
+
+Served alongside the star rating rework, these changes aim to accurately reflect the new sentiments of the star rating system.
+
+### Globally applied changes
+
+The final step of performance points calculation involves applying a simple multiplier to the calculated value, which consists of both difficulty and accuracy performance points. This allows for simple balancing changes, and works alongside specific multipliers found within their respective separate difficulty values.
+
+- The HD multiplier has been decreased from **1.10x** to **1.075x**, as a further bonus was placed in difficulty pp, rather than globally.
+- An EZ multiplier of **0.975x** has been added, to further nerf a problematic mod, especially when paired with DT.
+- The NF multiplier has been removed — as osu!taiko's HP system is different, it is not required nor needed, which also helps prevent sandbaggers in lower ranks.
+
+As well as performance point multiplier changes, the community survey highlighted issues with short maps being either underweighted at a full combo, or overweighted on plays with several misses.
+
+To combat this, misses will now significantly affect difficulty pp on maps with a max combo below 1000, scaling the miss count by a multiplier of `1000 / total successful hits`, and as such can significantly impact short maps with a higher miss count.
+
+![Effective miss count graph](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/effective-miss-count.jpg)
+
+### Changes to difficulty performance points values
+
+The following changes should result in impressive scores on harder beatmaps rewarding a significant increase of performance point values, with top player [syaron105](https://osu.ppy.sh/users/8741695) gaining over 6400 total pp!
+
+Please note that these changes only affect the difficulty part of performance, which refers to the star rating value under a mathematically scaled formula.
+
+- A further EZ multiplier of **0.985x** has been added, further supplementing its nerf mentioned above.
+- A new HR multiplier of **1.05x** has been added, as a way to rebalance the changes found within accuracy pp.
+- Difficulty pp now decreases by the square of decreasing accuracy instead of linearly, which in English means a huge nerf to previously overweighted low-accuracy scores.
+
+The new accuracy scaling formula is shown below:
+  
+![Accuracy scaling graph](/wiki/shared/news/2022-08-25-changes-to-osu-taiko-star-rating-and-performance-points/difficulty-accuracy-scaling.jpg)
```

Comment:
the y axis is wrong, refer to [the comment i made in the superseded pr](https://github.com/ppy/osu-wiki/pull/7883#discussion_r956643533)

![image](https://user-images.githubusercontent.com/36758269/187051731-78fdbabb-68c8-4eaa-bdd5-86acdc83673a.png)

![image](https://user-images.githubusercontent.com/36758269/187051879-d6c33fe0-f4a4-4e72-a49b-9d7611522c1e.png)

also this definitely needs to have the linear graph too for comparison

Reply:
I'll have to confirm with LTCA on his changes, previously I understood the change as `return difficultyValue` -> `return difficultyValue * Math.Pow(score.Accuracy, 2.0)`, but this diff does seem more likely. I'll commit the changes once he confirms.

Reply:
I was given the formula 2 ^ accuracy for the graph previously, will get back once he confirms

Reply:
He has confirmed, attached is the graph with the correct formula. But the title might need a change as it was written when I had the wrong understanding of the nature of the change.
![acc-diff-scaling](https://user-images.githubusercontent.com/5846322/189648329-3527cfab-64e5-4a34-80b6-56fbfe688ace.png)

Reply:
Legend seems wrong, will fix it and commit after a title has been decided

Reply:
Title was determined to be okay, fixed and applied in df42a9724388ae7a2b61f4badd875225c52d578b